jxl/src/api/output.rs

```diff
@@ -127,15 +127,27 @@ impl<'a> JxlOutputBuffer<'a> {
     pub(super) fn write_from_rgb_f32(&mut self, r: &Image<f32>, g: &Image<f32>, b: &Image<f32>) {
         assert_eq!(r.size(), g.size());
         assert_eq!(r.size(), b.size());
-        assert_eq!(self.bytes_per_row, r.size().0 * 12);
         let (xsize, ysize) = r.size();
-        assert_eq!(ysize, self.num_rows);
-        for y in 0..ysize {
+
+        // Handle case where the rendered image is smaller than the allocated buffer
```

Comment:
Please revert this change, and make sure to create buffers of the correct size in the first place.

Reply:
Done